src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -577,11 +577,11 @@ func buildRedpandaFlags(
 			if err != nil {
 				log.Warn(err)
 			} else if ioProps != nil {
-				yaml, err := iotune.ToYaml(*ioProps)
+				json, err := iotune.ToJSON(*ioProps)
```

Comment:
JSON instead of Json was required by the linter (but I guess it did not have a similar rule for YAML!).

Reply:
Yeah, I was surprised to see that we had Yaml instead of YAML since we enforce JSON, URL, API, etc...